data/coalition/coalition outfits.txt

```diff
@@ -239,15 +239,15 @@ outfit "Refueling Module"
 
 outfit "Shield Refactor Module"
 	category "Systems"
-	cost 2843000
+	cost 8324000
```

Comment:
I don't think this is needed. 2.8M for a 7t outfit is already high enough that it stands out as "Okay, this is clearly something really fancy and impressive".